HHS/Head-Start-TTADP
# [TTAHUB-1131] if the last goal is removed, the goal summary should display

## Description of change
The goal form is reopened if the last goal is removed.

## How to test
- complete activity summary
- on Goals and objectives, complete the form adding a goal and objective
- Click save and continue to view the goal summary
- Remove the goal
- the Goal summary should display in the default state with the "Select recipient's goal" field displayed 


## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1131

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
